test/Sentry.Log4Net.Tests/SentryAppenderTests.cs

```diff
@@ -291,6 +309,32 @@ public void Append_ConfiguredEnvironment()
             .CaptureEvent(Arg.Is<SentryEvent>(e => e.Environment == expected));
     }
 
+    [Fact]
+    public void MinimumEventLevel_DefaultsToNull()
+    {
+        var appender = new SentryAppender();
+        Assert.Null(appender.MinimumEventLevel);
+    }
+
+    [Fact]
+    public void MinimumEventLevel_ConvertsEventsBelowToBreadcrumbs()
```

Comment:
It'd be great to add two more unit tests to be explicit about what we expect:
1.  When `MinimumEventLevel is null`
2. When the Level is above `MinimumEventLevel` (make sure an event gets created in that case)

Reply:
Added `DoAppend_NullMinimumEventLevel_AddsEvent` and `DoAppend_AboveMinimumEventLevel_AddsEvent`, as in both suggested cases an event is sent.

Reply:
Perfect!